conn/bind_std.go

```diff
@@ -231,7 +231,7 @@ func (s *StdNetBind) receiveIP(
 	conn *net.UDPConn,
 	rxOffload bool,
 	bufs [][]byte,
-	sizes []int,
+	sizes [][2]int,
```

Comment:
:question: Confused: Is this tuple always going to be a range? Would it be more clear as a `type size struct{ start, end int }` or is that worse somehow?

Reply:
Practically speaking I believe the size and layout would be the same. I agree that a struct is more readable. We are also balancing with limiting change (literal LoC) to reduce drift from upstream before larger refactors are appropriate.